content/token-program.md

```diff
@@ -103,6 +103,8 @@ async function buildCreateMintTransaction(
 
 When manually building the instructions to create a new token mint, make sure you add the instructions for creating the account and initializing the mint to the *same transaction*. If you were to do each step in a separate transaction, it's theoretically possible for somebody else to take the account you create and initialize it for their own mint.
 
+You can refer to `createMint` documentation [here](https://solana-labs.github.io/solana-program-library/token/js/modules.html#createMint).
```

Comment:
Please [avoid 'here' links](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/). Instead use:

You can refer to the [`createMint` documentation](https://solana-labs.github.io/solana-program-library/token/js/modules.html#createMint).